src/watchmaker/workers/salt.py

```diff
@@ -131,7 +131,10 @@ def __init__(self, *args, **kwargs):
         self.salt_debug_log = kwargs.pop('salt_debug_log', None) or ''
         self.salt_content = kwargs.pop('salt_content', None) or ''
         self.salt_content_path = kwargs.pop('salt_content_path', None) or ''
-        self.ou_path = kwargs.pop('ou_path', None) or ''
+        self.ou_path_required = kwargs.pop('ou_path_required', False)
+        self.ou_path = kwargs.pop('ou_path', None) if not self.ou_path_required else kwargs.pop('ou_path', None)
```

Comment:
No need to change this
```suggestion
        self.ou_path = kwargs.pop('ou_path', None) or ''
```